app/controllers/v2/content_items_controller.rb

```diff
@@ -12,6 +14,18 @@ def index
       ).call
     end
 
+    def linkables
```

Comment:
I'm still struggling with the term `linkables` – can anyone think of something more descriptive?